homeassistant/components/weather/__init__.py

```diff
@@ -110,6 +118,10 @@ def state_attributes(self):
         if attribution is not None:
             data[ATTR_WEATHER_ATTRIBUTION] = attribution
 
+        forecast = self.forecast
+        if forecast is not None:
+            data[ATTR_FORECAST] = forecast
```

Comment:
Yes probably, I'll change later tonight.

The data structure returned by the openweathermap platform is the following:
```
[
  {
    'datetime': 'datetime',
    'temp': 'temperature'
  },...
]
```
If we keep this structure across all new and existing weather platforms I think it should be fairly easy to process this.
The current version of the frontend (that's in the PR) works with this data structure. The only culprit I've noticed with this is the fact that it shows up as `[Object object]` in some places.